datafusion/optimizer/src/push_down_filter.rs

```diff
@@ -792,29 +786,32 @@ mod tests {
     };
     use std::sync::Arc;
 
-    fn assert_optimized_plan_eq(plan: &LogicalPlan, expected: &str) -> Result<()> {
-        let optimized_plan = PushDownFilter::new()
-            .try_optimize(plan, &OptimizerContext::new())
-            .unwrap()
-            .expect("failed to optimize plan");
-        let formatted_plan = format!("{optimized_plan:?}");
-        assert_eq!(plan.schema(), optimized_plan.schema());
-        assert_eq!(expected, formatted_plan);
-        Ok(())
+    fn assert_optimized_plan_equal(plan: &LogicalPlan, expected: &str) -> Result<()> {
```

Comment:
Why do we rename the function here ?

Reply:
Because name exist conflict😂

Reply:
Oh, I see

Reply:
This works for me, although this is a nit:
```rust
    fn assert_optimized_plan_eq(plan: &LogicalPlan, expected: &str) -> Result<()> {
        crate::test::assert_optimized_plan_eq(Arc::new(PushDownFilter::new()), plan, expected)
    }
```

Reply:
Thank you @HaoYang670